### DISCUSSION TITLE
It doesn't work, I edit it but there is no change.

### DISCUSSION THREAD

**Discussion by knfe2000** [woocommerce/woocommerce#47747] [Checkout Flow]
It doesn't work, I edit it but there is no change.

**Comment by ralucaStan** [comment]
@Kawes1918 sounds like your dealing with some issues. It's helpful to provide more details about is not working for you.

- what page/block are you editing?
- what setting doesn't work for you?
- what are the steps you are doing? is the issue happening every time?
